backend/src/main/java/com/redhat/cloud/notifications/processors/email/RecipientResolver.java

```diff
@@ -19,7 +20,21 @@
     @Inject
     RbacRecipientUsersProvider rbacRecipientUsersProvider;
 
+    @ConfigProperty(name = "processor.email.rbac-user-query", defaultValue = "false")
+    Boolean rbacUserQuery;
```

Comment:
Nitpick: the `Boolean` will be unboxed each time it is used. That property will be used a lot so you could use a `boolean` here to avoid the cost of the unboxing.